static/css/components.css

```
@@ -761,7 +828,7 @@ button:disabled {
 
 /* ── Code / pre blocks ──────────────────────────────────── */
 code, pre {
-    font-family: 'JetBrains Mono', 'Fira Code', monospace;
+    font-family: JetBrains Mono, Fira Code, monospace;
```

Stylelint (17.12.0)

[error] Expected quotes around "JetBrains Mono" (font-family-name-quotes)

(font-family-name-quotes)